src/SourceTokenListIndex.ts

```diff
@@ -21,7 +21,7 @@ export default class SourceTokenListIndex {
     if (newIndex < 0 || this._sourceTokenList.length < newIndex) {
       return null;
     }
-    return this._sourceTokenList._getIndex(newIndex);
+    return this._sourceTokenList['_getIndex'](newIndex);
```

Comment:
Interesting that this type of access bypasses the private check but still gets the type right. Looks pretty reasonable to me.